### DISCUSSION TITLE
Can abuse report tickets be accessed? If so, how?

### DISCUSSION THREAD

**Discussion by roke-julian-lockhart** [community/community#136122] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I have submitted a report about a user,  but am unable to access that report at [`support.github.com/tickets/personal/0`](https://support.github.com/tickets/personal/0). Where can this report be accessed?

**Comment by GithmiHashara** [comment]
If you're unable to access the report through the link provided, it's possible that there may be an issue with your account permissions or the link itself. Here are a few things you can try:

Check the Correct Link: Ensure that you’re using the correct link to access your support tickets. The link should be https://support.github.com/tickets/personal for personal tickets.

Login Status: Make sure you are logged into the correct GitHub account that submitted the report. If you're logged in with a different account, the report may not be visible.

Browser Cache: Clear your browser cache or try accessing the link in an incognito/private window to rule out any caching issues.

Check Email for Confirmation: GitHub usually sends a confirmation email when a report is submitted. Check your inbox or spam folder for any emails from GitHub Support that may include a direct link to your ticket.

Contact GitHub Support: If none of the above solutions work, you might need to reach out to GitHub Support directly for assistance. You can contact them through https://support.github.com/contact.

**Reply by roke-julian-lockhart** [reply]
> [`#discussioncomment-10367312`](https://github.com/orgs/community/discussions/136122#discussioncomment-10367312)

@GithmiHashara:

1.	> If you're unable to access the report through the link provided, it's possible that there may be an issue with your account permissions or the link itself. Here are a few things you can try:
	> 
	> Check the Correct Link: Ensure that you’re using the correct link to access your support tickets. The link should be [`support.github.com/tickets/personal`](https://support.github.com/tickets/personal) for personal tickets.
	> 
	> Login Status: Make sure you are logged into the correct GitHub account that submitted the report. If you're logged in with a different account, the report may not be visible.
	
	I've filed multiple reports about multiple users during my tenure on GitHub, and they were never accessible via [`support.github.com/tickets/personal`](https://support.github.com/tickets/personal). Solely tickets created via [`support.github.com/request`](https://support.github.com/request) were.

	---
	
1.	> Check Email for Confirmation: GitHub usually sends a confirmation email when a report is submitted. Check your inbox or spam folder for any emails from GitHub Support that may include a direct link to your ticket.
	
	Indeed, I receive a confirmation message via e-mail. However, unlike most ZenDesk tickets, it doesn't include a URI to the portal:
	
	<blockQuote>
	
	```HTML
	<table style="width:100%;">
	        <tbody>
	                                    <tr>
	                <td style="padding:10px 0;width:100%;">
	                    <!DOCTYPE html><html><head>
	  <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
	  <style type="text/css">
	    table td {
	      border-collapse: collapse;
	    }
	  </style>
	
	          <style type="text/css">
	            @media only screen and (max-width: 768px) {
	              .simplified-email-footer .namecard {
	                display: block;
	                min-width: 100%;
	                padding: 0 0 16px 0; }
	
	              .simplified-email-footer .content {
	                padding: 16px; }
	            }
	          </style>
	        </head>
	<body style="width: 100%!important; margin: 0; padding: 0;">
	  <div style="padding: 10px ; line-height: 18px; font-family: 'Lucida Grande',Verdana,Arial,sans-serif; font-size: 12px; color:#444444;">
	    <div style="color: #b5b5b5;">## Please do not write below this line ##</div>
	    <p dir="ltr">  <br>    Thank you for contacting GitHub Support. We wanted to let you know that we've received your message. We are experiencing high volumes and therefore, you may experience longer than normal wait times. In the meantime, you may find answers to commonly asked questions in our <a href="https://github.community/">community forum</a> or in our <a href="https://docs.github.com/">documentation</a>.<br>  </p><p dir="ltr">  <br>    Ticket ID: 2944343<br>  <br>  </p><p dir="ltr">  <br>    If you have any additional information or would like to add anything to your initial message, now would be a great time to do so, feel free to reply to this email. If not, then rest assured your request is in the right hands :)<br>  </p><p dir="ltr">Thank you!<br>The GitHub Support Team</p>
	    <div style="color: #aaaaaa; margin: 10px 0 14px 0; padding-top: 10px; border-top: 1px solid #eeeeee;">
	      This email is a service from GitHub Support.
	    </div>
	  </div>
	<span style="color:#FFFFFF" aria-hidden="true">[1Z53JP-5LP6E]</span>
	
	<div itemscope="" itemtype="http://schema.org/EmailMessage" style="display:none">  <div itemprop="action" itemscope="" itemtype="http://schema.org/ViewAction">    <link itemprop="url" href="https://enterprise.githubsupport.com/hc/requests/2944343">    <meta itemprop="name" content="View ticket">  </div></div></body></html>
	                </td>
	            </tr>
	                            <tr>
	    <td>
	        <div style="margin:0px auto;max-width:896px;padding:10px 20px;background-color:#f5f7fa;text-align:center;line-height:1.5;font-size:12px;width:100%;border-left: 3px solid #19216c;font-family: ui-sans-serif, system-ui, -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, 'Helvetica Neue', Arial, 'Noto Sans', sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji', 'Segoe UI Symbol', 'Noto Color Emoji';color:#323f4b;overflow-wrap:break-word;">
	            This email was sent to <span style="font-weight:500;color:#19216c;">wy46xghh@rokejulianlockhart.addy.io</span> (https://github.com/settings/emails) from <span style="font-weight:500;color:#19216c;">developer@githubsupport.com</span><br>Click <a href="https://app.addy.io/deactivate/7dfa377d-94fe-495e-a2ba-25d815be2d6d?signature=94f589cb5828ec95aa7ae8062bf671dc0a2e42c7b8f4e2ff2475c762f494c49d" style="color:#2d3a8c;text-decoration:underline;" target="_blank" rel="noreferrer noopener nofollow">here</a> to deactivate this alias
	        </div>
	    </td>
	</tr>                    </tbody>
	    </table>
	```
	
	</blockQuote>

	---
	
1.	> Contact GitHub Support: If none of the above solutions work, you might need to reach out to GitHub Support directly for assistance. You can contact them through [`support.github.com/contact`](https://support.github.com/contact).
	
	Done so: [`support.github.com/ticket/personal/0/2944392`](https://support.github.com/ticket/personal/0/2944392). Thank you.

**Reply by roke-julian-lockhart** [reply, marked as answer]
> [`#discussioncomment-10367328`](https://github.com/orgs/community/discussions/136122#discussioncomment-10367328)

Per [`2944392#tc-29532863721748`](https://support.github.com/ticket/personal/0/2944392#tc-29532863721748:~:text=Tickets%20that%20are%20created%20from,so%20they%20can%20assist%20directly.):

<blockquote>

Tickets that are created from the report feature on [`GitHub.com`](http://github.com/) will not appear with regular tickets created on our support portal. This is because of their different nature, and because they're handled by a different team. 
 
If you'd like to follow up with a report about a user or content, you can respond directly to the e-mail notification. If you have any other questions, I could also transfer this ticket to our trust & safety team so they can assist directly.

</blockquote>
